src/main/scala/com/suprnation/actor/debug/TrackingActor.scala

```diff
@@ -142,9 +129,13 @@ final case class TrackingActor[F[+_]: Parallel: Concurrent: Temporal, Request, R
     messageBufferRef: Ref[F, Queue[Any]],
     restartMessageBufferRef: Ref[F, Queue[(Option[Throwable], Option[Any])]],
     errorMessageBufferRef: Ref[F, Queue[(Throwable, Option[Any])]],
+    timerGenRef: Ref[F, Int],
+    timersRef: Ref[F, Timers.TimerMap[F, String]],
     proxy: ReplyingActor[F, Request, Response]
 ) extends ReplyingActor[F, Request, Response]
-    with ActorConfig {
+    with ActorConfig
+    with Timers[F, Request, Response, String] {
```

Comment:
Hardcoded the timer `Key` to `String` to avoid having an extra type param we won't care about most times